package.json

```diff
@@ -9,8 +9,15 @@
     "start": "next start",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
-    "lint": "eslint . --ext=tsx,ts",
-    "test": "yarn lint"
+    "lint": "eslint . --ext=tsx,ts,js,jsx",
+    "test": "is-ci \"test:coverage\" \"test:watch\"",
```

Comment:
Id recommend not using `is-ci`. Just have `test` be reserved for 1 time running, not watching. Developers can use a separate script for watching

Reply:
We actually not want this to be `yarn link; yarn test:coverage`, no?

Reply:
so `test` script should just run `jest`, right?

Reply:
I am not familiar with `yarn link`, can you explain what you mean by running `yarn link; yarn test:coverage`

Reply:
you want us to have a script that will lint and runs the tests you mean?
maybe we can add a `validate` script that will do that

Reply:
yep. we want CI to run `npm run test` that will run lint and unit tests.

Reply:
If we run `npm run test` it will check both tests and lint errors since I configured jest with eslint